min_opening: 0.0
max_opening: 0.0316
tolerance: 0.0005
joint: phantomx_pincher_gripper_finger1_joint
